prediction-applier-extension/src/main/java/org/alfresco/hxi_connector/prediction_applier/rest/api/model/PredictionModel.java

```diff
@@ -0,0 +1,73 @@
+/*-
+ * #%L
+ * Alfresco HX Insight Connector
+ * %%
+ * Copyright (C) 2023 - 2024 Alfresco Software Limited
+ * %%
+ * This file is part of the Alfresco software.
+ * If the software was purchased under a paid Alfresco license, the terms of
+ * the paid license agreement will prevail.  Otherwise, the software is
+ * provided under the following open source license terms:
+ *
+ * Alfresco is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Lesser General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Alfresco is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with Alfresco. If not, see <http://www.gnu.org/licenses/>.
+ * #L%
+ */
+package org.alfresco.hxi_connector.prediction_applier.rest.api.model;
+
+import java.io.Serializable;
+import java.util.Date;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+
+import org.alfresco.hxi_connector.prediction_applier.service.model.Prediction;
+
+@Accessors(prefix = {"_", ""})
+@Getter
+@Setter
+@AllArgsConstructor
+@NoArgsConstructor
+@SuppressWarnings("PMD.FieldNamingConventions")
+public class PredictionModel
+{
+    private String _id;
```

Comment:
Why property name does starts with the underscore? 🤔

Reply:
Oh, ok - I didn't know about this feature 😅

Reply:
Strange. Either there's something hidden in WebScripts guts or I am missing something but I see `_id` used in constructor of `Prediction` here in `toServiceModel()` method which is used by `PredictionServiceImpl::appyPredictions` method which overwrites `id` anyway.
To me, this is somehow "magic number" approach and would need at least a comment or checking whether this is actually a valid case.

Reply:
Investigating further, the reason this works is because it is a Jackson feature:
https://github.com/FasterXML/jackson/blob/master/contribution/jackson-coding-style-guide.md#field-naming